Enigmatry.Entry.TemplatingEngine.Fluid/Enigmatry.Entry.TemplatingEngine.Fluid.csproj

```diff
@@ -1,22 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-        <TargetFramework>net8.0</TargetFramework>
-        <ImplicitUsings>enable</ImplicitUsings>
-        <Nullable>enable</Nullable>
-        <RootNamespace>Enigmatry.Entry.TemplatingEngine.Liquid</RootNamespace>
```

Comment:
Formatting change, but I noticed this RootNamespace (talked with Laslo, it seems there is an error here). Will be solve later